style: apply /style-guide pass to models/runs (#2724)
wandb/docs: models/runs/alert.mdx

To set up scriptable alerts with `run.alert()`, take these steps, which are detailed in the following sections:

1. Turn on **Alerts** in your W&B [User Settings](https://wandb.ai/settings).
2. Add `run.alert()` to your code.
3. Test the configuration.

> ## Turn on alerts in your W&B user settings

Reviewer: ```suggestion
### Turn on alerts in your W&B user settings
```

Author: Applied — demoted this heading to `###`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

This step enables the alert delivery channels (Slack or email) that `run.alert()` calls use. You only have to do this the first time you set up W&B Alerts, or when you want to modify how you receive alerts.

In your [User Settings](https://wandb.ai/settings), configure the following:

* Scroll to the **Alerts** section.
* Turn on **Scriptable run alerts** to receive alerts from `run.alert()`.
* Use **Connect Slack** to pick a Slack channel to post alerts. W&B recommends the **Slackbot** channel because it keeps alerts private.
* **Email** goes to the email address you used when you signed up for W&B. W&B recommends setting up a filter in your email so that all these alerts go into a folder and don't fill your inbox.

<Frame>
    <img src="/images/track/demo_connect_slack.png" alt="Alerts settings in W&B User Settings"  />
</Frame>

After you complete these settings, W&B can deliver alerts to your chosen channels whenever your code calls `run.alert()`.

> ## Add `run.alert()` to your code

Reviewer: ```suggestion
### Add `run.alert()` to your code
```

Author: Applied — demoted this heading to `###`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

> This step defines when alerts fire and what they contain. Each `run.alert()` call defines the condition that triggers a notification and the content of the message you receive. Add `run.alert()` to your code (either in a Notebook or Python script) wherever you want to trigger it.

Reviewer: ```suggestion
This step sets the condition that triggers an alert and the message that the alert sends. Add `run.alert()` to your notebook or Python script where you want to send an alert.
```

Author: Applied your suggested rewording.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

    run.alert(title="High Loss", text="Loss is increasing rapidly")
```

> ## Test the configuration

Reviewer: ```suggestion
### Test the configuration
```

Author: Applied — demoted this heading to `###`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

> Confirm that alerts reach you before you rely on them in production runs. Run your code to trigger the alert, then check your Slack or emails for the alert message. If you don't receive any, make sure you have emails or Slack turned on for **Scriptable run alerts** in your [User Settings](https://wandb.ai/settings).

Reviewer: ```suggestion
Run your code to trigger the alert, then check Slack or your email for the alert message. If you don’t receive the alert, make sure email or Slack notifications are turned on for Scriptable run alerts in your [User Settings](https://wandb.ai/settings).
```

Author: Applied your rewording; normalized the apostrophe to straight ASCII to match the rest of the doc.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

        )
```

## Tag or mention users

Tag users to direct alerts to the right person and trigger Slack's standard mention notifications. To tag yourself or a colleague, use the at sign `@` followed by the Slack user ID in the alert title or text. You can find a Slack user ID from their Slack profile page.

```python
run.alert(title="Loss is NaN", text=f"Hey <@U1234ABCD> loss has gone to NaN")
```

## Configure team alerts

> Besides individual alerts, team admins can configure alerts that apply to everyone on the team. Team admins can set up team alerts on the team settings page: `wandb.ai/teams/[YOUR-TEAM]`.

Reviewer: ```suggestion
In addition to individual alerts, team admins can configure alerts that apply to everyone on the team. Team admins can set up team alerts on the team settings page: `wandb.ai/teams/[YOUR-TEAM]`.
```

Author: Applied — changed "Besides" to "In addition to".

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
